### DISCUSSION TITLE
mpd process command "idle" and "idle "player"" returned 1

### DISCUSSION THREAD

**Discussion by vojislav** [MusicPlayerDaemon/MPD#1504] [Q&A] [answered]
I have a script that runs every second in i3bar and gets the current song and time using mpc. Sometimes, every 15 minutes or so, while playing or paused, it doesn't produce any text. I ran `mpc status` when it happened and the output is as if mpd is in idle state
```
$ mpc status
volume:100%   repeat: off   random: off   single: off   consume: off
```
even though a song is playing. After a few second, it starts working again.

I set the mpd log_level to "verbose" and checked the log when it happened, [here's the relevant part of the log](https://paste.debian.net/hidden/9b183c23/), and it shows running command "status" and "currentsong" which return 0, but there's also `process command "idle"` and `process command "idle "player""` which return 1. I looks like it loses connection to mpd for a few seconds.

```
$ mpc version
mpd version: 0.21.4
```
[mpd --version](https://paste.debian.net/1236421/)
[mpd.conf](https://paste.debian.net/hidden/811afcf3/)

**Comment by jcorporation** [comment]
return 1 for the idle command is no error condition, as far I know.

**Comment by MaxKellermann** [comment, marked as answer]
Your MPD version is very old and unsupported. See https://www.musicpd.org/help/

**Reply by vojislav** [reply]
updating mpd to 0.23.6 seems to have fixed it. thanks!
